solidus_reports.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
     s.metadata["source_code_uri"] = s.homepage
   end
 
-  s.required_ruby_version = "~> 2.4"
+  s.required_ruby_version = [">= 2.4", "< 3.2"]
```

Comment:
Would it make sense to open the requirement to every 3.x?

This is the current value in solidus_dev_support I think:

```
Gem::Requirement.new('>= 2.5', '< 4')
```